### DISCUSSION TITLE
Can Argocd can generate a set of ApplicationSets, not just Application?

### DISCUSSION THREAD

**Discussion by byvinx** [argoproj/argo-cd#13603] [General]
Hi, using ApplicationSets I saw how it is possible to generate a set of Applications starting for example with git generator from a git repository. In Argocd is there the possibility to generate ApplicationSet, and not only Application, based on information present on a git repo? It might look like an ApplicationSetSet :)

**Comment by jaideepr97** [comment]
Hi @byvinx 
I don't think this is possible using Argo CD at the moment 

you could probably do this using kustomize overlays though
